packages/venia-concept/templates/default-font-include.mst

```diff
@@ -0,0 +1,30 @@
+{{!
+  Inline the contents of the Google Muli font to avoid initial blocking load
+  via <link>. Apparently css urls are asynchronous.
+}}
+<style type="text/css">
+  /* vietnamese */
+  @font-face {
+    font-family: 'Muli';
+    font-style: normal;
+    font-weight: 300;
+    src: local('Muli Light'), local('Muli-Light'), url(https://fonts.gstatic.com/s/muli/v13/7Au_p_0qiz-adZnkOCr2z24PMFk-0g.woff2) format('woff2');
+    unicode-range: U+0102-0103, U+0110-0111, U+1EA0-1EF9, U+20AB;
```

Comment:
Seems like all these unicode ranges could be combined, since they're all the exact same otherwise.

Reply:
I'm new to unicode ranges so I'll have to read into what that means but I do want to say that I copied this directly from the API response for the google font.

If I customize this I'll want to add a comment saying as such so that if we go with a different font whomever updates it does the same thing.